### PR TITLE
Eapeak changes

### PR DESCRIPTION
**Pull request by jacobj10**
Sum up major changes:

Connection much more reliable (used srp1() instead of sendp() sniff() combination for auth and assoc req)

eapscan working and tested up to legacy_nak (was not able to configure radius server to respond to identity response, therefore cannot confirm the identity response is received and responded to, but it should)

BSSID filter suported for eapeak

Possible problem with packet sent to receive rsnInfo identified (possibly use srp1() and change contents of packet)

Deleted eapwn

Cleaned up, renamed variables, styled according to PEP 8

No current add'l exception handling discovered, but will keep looking

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -49,37 +35,34 @@
 
 setup(
 	name='EAPeak',
-	version='0.1.4',
+	version='0.1.5',
```
Comment (zeroSteiner) [inline setup.py]: This should actually be `0.1.6` like it is in eapeak. I'm guessing this file has been one version behind for a while.

**Comment by jacobj10** [comment]
Fixed srp1() to handle large network traffic with BPF filter

**Comment by jacobj10** [comment]
PR ready for testing!

**Comment by jacobj10** [comment]
Made methods.

**Comment by zeroSteiner** [comment]
Alright I just merged this in, thanks.

**Pull request merged**